tox.ini

```diff
@@ -84,6 +84,8 @@ commands_pre =
     # strictly only need to uninstall pytest-qt (which will raise without a backend)
     # the rest is for good measure
     headless: pip uninstall -y pytest-qt qtpy pyqt5 pyside2
+    # Install localization test package
+    headless: pip install napari/utils/_tests/napari-language-pack-es_CO
```

Comment:
@tlambert03 this is what I tried but maybe I should do this differently?

Help :) !

Reply:
probably should just put `napari/utils/_tests/napari-language-pack-es_CO` up in `deps`.

however, I'm trying to get translations to work locally on pytest and running into issues, here's what I tried, let me know what I should be doing:

```sh
pip install napari/utils/_tests/napari-language-pack-es_CO
...
Successfully installed napari-language-pack-es-CO-0.1.0
```

```python
In [1]: from napari.utils.translations import translator

In [2]: translator
Out[2]: napari.utils.translations.translator

In [3]: translator._set_locale("es_CO")

In [4]: trans = translator.load()

In [5]: trans
Out[5]: <napari.utils.translations.TranslationBundle at 0x7fea400986d0>

In [6]: trans._("MORE ABOUT NAPARI")
Out[6]: 'MORE ABOUT NAPARI'
```

Reply:
Hmm yes, I think I forgot to update the MANIFEST, the right way. I was using 

```
pip install -e napari/utils/_tests/napari-language-pack-es_CO
```

Reply:
ahh, cool.  OK, yeah, once that's done, I think moving the tox install up to the deps section (instead of the `commands_pre` section) _should_ do it ... but might need a trick for a local path?

Reply:
Will test in a min.